trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -207,6 +207,11 @@ Platforms and libraries MUST expect header name in any casing and SHOULD send he
 
 ## Header value
 
+Multiple `tracestate` headers are allowed. Values from multiple headers in
+incoming requests SHOULD be combined in a single header according to the
```

Comment:
Why SHOULD?

Reply:
The idea is to increase interoperability by making sure it's always a single string with defined order of elements and easy to calculate length. From implementation perspective it is also easier if we talking about libraries, not generic proxies. 

What's your main concern with this "SHOULD"?

Reply:
In the HTTP protocol multiple headers and single header with comma-separated values are interchangeable. I don't see the point of putting more stringent restrictions, because a compliant HTTP middleware is free to swap those two representations.

Reply:
That's why it SHOULD, not MUST. Same as here:

> Platforms and libraries MUST expect header name in any casing and SHOULD send header name in lower case.

Reply:
This is why I object to this. If you MUST treat different inbound formats anyway, putting additional restrictions on the outbound format adds 0 value to anyone, just becomes an arbitrary thing in the spec with no justification.